<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(47, 39, 39);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>19</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	
	color: rgb(119, 118, 123);
}</string>
        </property>
        <property name="text">
         <string>Subtitle</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="list_ShowSubtitle">
          <property name="styleSheet">
           <string notr="true">QListWidget
{

	
	color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(36, 31, 49);
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">QLabel{
	
	color: rgb(119, 118, 123);
}</string>
             </property>
             <property name="text">
              <string>Text</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="edt_text">
             <property name="styleSheet">
              <string notr="true">QTextEdit{
	
		
	color: rgb(255, 255, 255);

}</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edt_start">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	
	color: rgb(255, 255, 255);
}</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">QLabel{
	
	color: rgb(119, 118, 123);
}</string>
             </property>
             <property name="text">
              <string>End time:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edt_end">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	
	color: rgb(255, 255, 255);
}</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">QLabel{
	
	color: rgb(119, 118, 123);
}</string>
             </property>
             <property name="text">
              <string>Start time:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_Save">
            <property name="minimumSize">
             <size>
              <width>521</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	
	color: rgb(61, 56, 70);
	background-color: rgb(111, 63, 26);
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStart">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	
	color: rgb(61, 56, 70);
	background-color: rgb(111, 63, 26);
}</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="btn_negetive">
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	
	color: rgb(61, 56, 70);
	background-color: rgb(111, 63, 26);
}</string>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edt_shifttime">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	
	color: rgb(255, 255, 255);
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_plus">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	
	color: rgb(61, 56, 70);
	background-color: rgb(111, 63, 26);
}</string>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btn_change">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	
	color: rgb(61, 56, 70);
	background-color: rgb(111, 63, 26);
}</string>
               </property>
               <property name="text">
                <string>Change</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
